pages/news/2020-05-22-why-mixxx-will-not-support-streaming.html

```diff
@@ -0,0 +1,84 @@
+If a user user didn't want to do this on their own, it would be easy for anyone  to do it for them and host downloads of modified versions of Mixxx that could record and broadcast from streaming services.
+It would be difficult for us to distance these unofficial modifications from Mixxx, which would reflect badly on Mixxx.
+Other software teams [already](https://kodi.tv/article/piracy-box-sellers-and-youtube-promoters-are-killing-kodi) [have](https://kodi.tv/article/kodi-builds-streaming-add-ons-and-free-lunches) [trouble](https://kodi.tv/article/is-kodi-legal) [distancing their name](https://kodi.tv/article/in-defence-of-our-good-name) from illegal third party modifications/addons.
+Without supporting streaming, Mixxx already has [enough trouble](/news/2020-05-22-you-dont-need-to-pay-for-mixxx/) with third parties selling copies of Mixxx without complying with the license or trademark or providing support.
+We do not want to make this problem worse.
+Mixxx is made by a small team of volunteers (we are always looking for new people to [join us](/get-involved)!).
+We want to spend our time making Mixxx better, not defending the reputation of our work from unscrupulous third parties.
+
+There are many practical reasons to not support streaming services as well.
+Supporting streaming services in Mixxx would require a lot of work to implement, perhaps months of work.
+As with all Mixxx features, this would be done entirely by volunteers with limited time.
+Those volunteers would not be using that time working on other features.
+
+All this hard work could be wasted at the whim of the streaming company or a record company they get music from. 
+Either the streaming company or record company could decide to no longer allow Mixxx to use the streaming service or they could go out of business.
+These are not hypothetical concerns.
+They have already happened several times in the short history of DJ software using streaming services.
+Spotify [stopped VirtualDJ from using their service](https://djtechtools.com/2017/09/14/spotify-axes-dj-software-streaming-virtual-dj/) as well as [Algoriddim djay](https://djworx.com/spotify-withdraws-from-dj-streaming-apparently-not-the-end-of-the-world/).
+PulseLocker [shut down without any notice](https://djtechtools.com/2017/11/10/pulselocker-folds-streaming-dj-software/) after Serato, Rekordbox, and Virtual DJ had integrated the service.
+
+As a DJ, relying on streaming is not a good idea either.
+Internet connections will never be as reliable as music stored on your computer.
+Even if your home connection is reliable, that does not mean the Internet connection at the venue you play at will be reliable.
+Furthermore, you could lose access to the streaming service at the whim of the streaming or record company as discussed above, if you no longer can afford the subscription, or if you don't automatically renew the subscription fee.
+This could even happen during your gig without any warning.
```

Comment:
```suggestion
Furthermore, you could lose access to the streaming service at the whim of the streaming or record company as discussed above, or if you simply don't renew your subscription - this could even happen during a gig without any warning!
```